### DISCUSSION TITLE
(Duplicate of #5205)

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#5218] [Q&A] [answered]
My issue is back )-:

https://github.com/netblue30/firejail/issues/5205

**Comment by kmk3** [comment, marked as answer]
@osevan [on Jun 24](https://github.com/netblue30/firejail/discussions/5218#discussion-4171206):

> My issue is back )-:
> 
> #5205

Then just reopen the issue instead of spamming it here.

**Reply by osevan** [reply]
Already done
